### DISCUSSION TITLE
Denial of service. Css-what

### DISCUSSION THREAD

**Discussion by SteveTenadjang** [tailwindlabs/tailwindcss#4793] [Help]
![IMG_20210624_115511_695](https://user-images.githubusercontent.com/46106643/123251625-52fe6300-d4e3-11eb-8bf3-39252db26ccb.jpg)

Please for help😣

**Comment by MartinHughes-BPC** [comment]
Same here, I expect it'll get patched soonish. I don't worry *too* much about vulnerabilities in laravel-mix as I only use it on my dev machine(s) to compile the css and js assets I then push to production. If it was running on my production server (or indeed another package in production relied on a vulnerable package) I'd worry a lot more.

**Reply by SteveTenadjang** [reply]
https://docs.npmjs.com/cli/v6/commands/npm-audit

**Reply by SteveTenadjang** [reply]
Check the link above that's where i found my solution
